sdk/search/azure-search-documents/azure/search/documents/indexes/aio/_search_index_client.py

```diff
@@ -603,3 +604,16 @@ async def create_or_update_alias(
             alias_name=alias.name, alias=alias, prefer="return=representation", error_map=error_map, **kwargs
         )
         return result  # pylint:disable=protected-access
+
+    @distributed_trace_async
+    async def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs) -> AsyncHttpResponse:
```

Comment:
Noticed that the generated client's `_send_request` method uses `Awaitable[AsyncHttpResponse]` as the typing for the return value. Should we follow suit?

Reply:
I followed this one: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_form_base_client_async.py#L94 :)

Reply:
ah, realized that you are awaiting the internal _send_request below, so the typing makes sense here.